_includes/payee-info.md

```diff
@@ -1,4 +1,20 @@
-{% assign instrument = include.payment_instrument | default: "paymentorder" %}
+{% assign instrument = include.payment_instrument | default: "paymentorders" %}
+{% case instrument %}
```

Comment:
This is awesome! But to simplify things, perhaps you can `{% assign length = 50 %}` before the `case` statement to give `length` the default value of `50` and then only have `when` statements for those payment instruments that have a `length` different from `50`?